### PR TITLE
Feature / Breadcrumbs

### PR DESCRIPTION
**Pull request by salesfelipe**
#### What is the purpose of this pull request?
Create the Breadcrumbs component

#### What problem is this solving?
Give to the user the ability to go back to a previous point

#### How should this be manually tested?
[workspace](https://felipe--storecomponents.myvtex.com/smartphones/s)

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by brunojdo** [comment]
https://github.com/vtex-apps/breadcrumb/blob/master/render/index.js

File: react/components/Breadcrumbs/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+/**
+ * Breadcrumb Component.
+ */
+export class Breadcrumb extends Component {
+  render() {
+    const { search, slug } = this.props
+
+    return (
+      <div className="vtex.breadcrumbs pa4">
```
Comment (ClaudivanFilho) [inline react/components/Breadcrumbs/index.js]: vtex-breadcrumbs

File: react/components/Breadcrumbs/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Breadcrumbs
```
Comment (ClaudivanFilho) [inline react/components/Breadcrumbs/README.md]: Melhor deletar esse README.md

File: react/components/Breadcrumbs/index.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'render'
+
+/**
+ * Breadcrumb Component.
+ */
+export class Breadcrumb extends Component {
+  render() {
+    const { search, slug } = this.props
```
Comment (ClaudivanFilho) [inline react/components/Breadcrumbs/index.js]: não seria melhor o breadcrumb receber uma pilha?

Comment (ClaudivanFilho) [reply react/components/Breadcrumbs/index.js]: ex: `[{name, page, params}, {name, page, params}]`

**Pull request closed without merging**